packages/plugins/i18n/admin/src/translations/en.json

```diff
@@ -25,8 +25,9 @@
   "Settings.locales.modal.advanced.setAsDefault.hint": "One default locale is required, change it by selecting another one",
   "Settings.locales.modal.advanced.settings": "Settings",
   "Settings.locales.modal.base": "Base settings",
+  "Settings.locales.modal.base.clearLabel": "Clear label",
   "Settings.locales.modal.create.alreadyExist": "This locale already exists",
-  "Settings.locales.modal.create.confirmation": "Add locale",
+  "Settings.locales.modal.create.confirmation": "Save",
```

Comment:
No need to recreate this one it already exists the id is : `app.components.Button.save`